pyramid/eviction.go

```diff
@@ -3,37 +3,57 @@ package pyramid
 import (
 	"fmt"
 
-	lru "github.com/treeverse/golang-lru"
-	"github.com/treeverse/golang-lru/simplelru"
+	"github.com/dgraph-io/ristretto"
 )
 
 // eviction is an abstraction of the eviction control for easy testing
 type eviction interface {
 	touch(rPath relativePath)
-	store(rPath relativePath, filesize int64) int
+	store(rPath relativePath, filesize int64) bool
```

Comment:
Can you document the return value?  It evidently has at least two possible interpretations, given that it changed.